### PR TITLE
site: Fixup codespell errors

### PR DESCRIPTION
**Pull request by stevesloka**
Found a bunch of codespell errors happening locally which ironically did
not show up in the CI build tests.

```
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACH ==> CATCH, CACHE
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACH ==> CATCH, CACHE
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACH ==> CATCH, CACHE
./site/public/css/style.css.map:52: CACH ==> CATCH, CACHE
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACH ==> CATCH, CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACH ==> CATCH, CACHE
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACH ==> CATCH, CACHE
./site/public/css/style.css.map:52: MAPE ==> MAP
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACH ==> CATCH, CACHE
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACH ==> CATCH, CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CAcL ==> calc
./site/public/css/style.css.map:52: CACH ==> CATCH, CACHE
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACH ==> CATCH, CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACH ==> CATCH, CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACH ==> CATCH, CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACH ==> CATCH, CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACL ==> CALC
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/css/style.css.map:52: CACE ==> CACHE
./site/public/docs/v1.16.0/configuration/index.html:1397: boostrap ==> bootstrap
./site/public/docs/v1.12.0/configuration/index.html:1023: boostrap ==> bootstrap
./site/public/docs/v1.15.0/configuration/index.html:1556: boostrap ==> bootstrap
./site/public/docs/v1.11.0/configuration/index.html:1016: boostrap ==> bootstrap
./site/public/docs/v1.15.1/configuration/index.html:1382: boostrap ==> bootstrap
./site/public/docs/v1.10.1/configuration/index.html:1009: boostrap ==> bootstrap
./site/public/docs/v1.14.0/configuration/index.html:1556: boostrap ==> bootstrap
./site/public/docs/v1.10.0/configuration/index.html:1009: boostrap ==> bootstrap
./site/public/docs/v1.14.1/configuration/index.html:1556: boostrap ==> bootstrap
./site/public/docs/v1.13.1/configuration/index.html:1438: boostrap ==> bootstrap
./site/public/docs/main/configuration/index.html:1397: boostrap ==> bootstrap
./site/public/docs/v1.13.0/configuration/index.html:1438: boostrap ==> bootstrap
./site/public/guides/xds-migration/index.html:74: boostrap ==> bootstrap
./site/public/fonts/README.md:7: millenium ==> millennium

```

Signed-off-by: Steve Sloka <slokas@vmware.com>

### CODE REVIEW

**Comment by sunjayBhatia** [comment]
"boostrap" 😂

File: .codespell.skip [hunk 1]
```diff
@@ -8,4 +8,5 @@
 *.ico
```
Comment (sunjayBhatia) [inline .codespell.skip:8]: probably need to update the CI job to add this too?

is there a way we can use this file as input to the job, that would make it easier to keep up to date

Comment (sunjayBhatia) [reply .codespell.skip:8]: can't see anything from a quick glance at the action README

**Review by youngnick — APPROVED** [review]

File: .codespell.skip (hunk 1 above)
Comment (stevesloka) [reply .codespell.skip:8]: updated the job. I don't see a way other than moving back to using the makefile for everything (which I'd personally prefer). This way there's no chance of skewing the jobs.

**Review by skriss — APPROVED** [review]

**Pull request merged**